fontforgeexe/prefs.c

```diff
@@ -313,6 +314,7 @@ static struct prefs_list {
 	{ N_("AutoSaveFrequency"), pr_int, &AutoSaveFrequency, NULL, NULL, '\0', NULL, 0, N_( "The number of seconds between autosaves. If you set this to 0 there will be no autosaves.") },
 	{ N_("RevisionsToRetain"), pr_int, &prefRevisionsToRetain, NULL, NULL, '\0', NULL, 0, N_( "When Saving, keep this number of previous versions of the file. file.sfd-01 will be the last saved file, file.sfd-02 will be the file saved before that, and so on. If you set this to 0 then no revisions will be retained.") },
 	{ N_("UndoRedoLimitToSave"), pr_int, &UndoRedoLimitToSave, NULL, NULL, '\0', NULL, 0, N_( "The number of undo and redo operations which will be saved in sfd files.\nIf you set this to 0 undo/redo information is not saved to sfd files.\nIf set to -1 then all available undo/redo information is saved without limit.") },
+	{ N_("UnicodeSupremacy"), pr_bool, &UnicodeSupremacy, NULL, NULL, '\0', NULL, 0, N_( "Whether or not the Unicode Standard should have supremacy even in cases its contents may be dubious, controversial, or even wrong.\nThis currently only changes certain build procedures when using “Make Accented Glyph”.") },
 	PREFS_LIST_EMPTY
```

Comment:
"Supremacy" is probably the wrong impression here, plus the word does conjure up negative impressions of racism. What you probably want to say here is have it default to Unicode standards, and enable backwards compatibility with George's edits for users that want to maintain backwards compatibility for these glyphs. You also want to indicate what you are modifying (like in this case it's combiners I think).
Maybe OldCombiners, "Enable backwards compatible combiner codes for users maintaining older fonts"
...probably not the best choice of words, but suggests we're defaulting to unicode values, and giving users a switch to enable older values if they need it.
I would prefer use pr_int instead of pr_bool for consistency, plus ints are able to convey error codes while bools cannot.
...just my 2 cents review